src/rpc/handlers/LedgerEntry.hpp

```diff
@@ -70,6 +70,7 @@ class LedgerEntryHandler {
         std::optional<boost::json::object> node;
         std::optional<std::string> nodeBinary;
         bool validated = true;
+        uint32_t deleted_ledger_index;
```

Comment:
should I give it a default value like 0?

Reply:
you should make it optional, this field should only be present if the object was deleted